react/components/InputSearch/index.js

```diff
@@ -94,7 +93,10 @@ class InputSearch extends Component {
                   InputSearch.separatorHeight.regular,
               }}
             />
-            <span className="pointer pl4 c-link" onClick={this.handleSubmit}>
+            <span
+              className="pointer pl4 c-link"
+              onClick={this.handleSubmit}
+              style={{ height: iconSize }}>
```

Comment:
What do you think?

```suggestion
              className="pointer pl4 c-link flex"
              onClick={this.handleSubmit}>
```

Reply:
This is because we prefer to use tachyons classes instead of the `style` prop when possible :)